reference-data/src/test/java/uk/gov/hmcts/payment/referencedata/controllers/ReferenceDataControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.payment.referencedata.controllers;
```

Comment:
This controller test is to be updated in the reference data package which already as test class.
We can remove here.